src/app/shared/services/account/account.service.ts

```diff
@@ -607,4 +607,9 @@ export class AccountService {
   private getStorage(key) {
     return this.storage.local.get(key) || this.storage.session.get(key);
   }
+
+  public updateAccountStorage(amount) {
```

Comment:
```suggestion
  public deductAccountStorage(amount: number) {
```
I think we should probably rename this method to make it more clear what's happening. My first thought was that `updateAccountStorage` would be a simple setter instead of using the `-=` operator. I think if we are using a relative setter it might be best to make it `addAccountStorage` instead, so that it's positive by default and only negative when we're taking things away. It's a bit confusing to pass a negative into the function if we're *adding* storage, but it would also just be clear if we named it `deductAccountStorage` instead.